api/requestParams.d.ts

```diff
@@ -142,6 +157,7 @@ export interface CatMaster extends Generic {
 export interface CatNodeattrs extends Generic {
   format?: string;
   local?: boolean;
+  cluster_manager_timeout?: string;
   master_timeout?: string;
```

Comment:
I believe we can use JSDoc here:
for example:
```
cluster_manager_timeout?: string;
/**
 * @deprecated use cluster_manager_timeout instead
 */
master_timeout?: string;
```

Reply:
Oh I see @joshuarrrr, made similar comments. Yes, please explicitly add the `@deprecated` annotation to each deprecated field and function.

Reply:
Here is the benefit:
![Screen Shot 2022-05-05 at 11 50 14 PM](https://user-images.githubusercontent.com/12060704/167081334-531160c5-ef88-4a85-a4bc-6ae87615b683.png)